speechbrain/lobes/models/transformer/Conformer.py

```diff
@@ -385,12 +641,55 @@ def forward(
                 src_mask=src_mask,
                 src_key_padding_mask=src_key_padding_mask,
                 pos_embs=pos_embs,
+                dct_config=dct_config,
             )
             attention_lst.append(attention)
         output = self.norm(output)
 
         return output, attention_lst
 
+    def forward_streaming(
```

Comment:
Docstring and difference as well ;-) Is it very necessary for this class to have a custom forward? I see many changes for the EncoderLayer, so it makes sense, but here?

Reply:
I guess the question in general is whether the streaming forward functions should be named `forward_streaming` (and that the streaming behavior would be triggered by having the context object non-`None`).

Part of the problem is that it needs to call into layers' `forward_streaming` and not merely the forward as well, and that, down the road, other encoders may need to do something more involved that would justify a separate forward.

TBH, I think it's better this way for now.